packages/css/src/components/card/card.scss

```
@@ -50,8 +30,23 @@
   text-decoration-thickness: var(--ams-card-link-text-decoration-thickness);
   text-underline-offset: var(--ams-card-link-text-underline-offset);
 
+  // Make the whole Card clickable by making the link cover its entire area.
+  // Source: https://inclusive-components.design/cards/
+  &::after {
+    content: "";
+    display: block;
+    inset-block: 0;
+    inset-inline: 0;
+    position: absolute;
+  }
+
   &:hover {
     color: var(--ams-card-link-hover-color);
     text-decoration-line: var(--ams-card-link-hover-text-decoration-line);
   }
+
+  // Remove the outline from the link; it’s already on the Card.
```

I think we should mention the reason for the `:has` selector here. Line 14 and this line work in tandem. 

```suggestion
  // Remove the outline only if :has is supported,
  // which means the outline on the whole card is set
```

---

Alright, but I still rephrased it.